apps/_documentation/static/js/markmin.js

```diff
@@ -27,7 +27,7 @@ jQuery.fn.markmin = (function(){
         var re_nn = /\n\s*\n/g;
         var re_strong = /\*\*([^\s*][^*]*[^\s*])\*\*/g;
         var re_em = /''([^\s\'][^\']*[^\s\'])''/g;
-        var re_blockquote = /[-]{5}[-]*\n(.*?)\n[-]{5}[-]*/gm;
+        var re_blockquote = /[-]{5}[-]*([\s\S]*?)[-]{5}[-]*/gm;
```

Comment:
A blockquote including a new line character was not being properly rendered. It was displayed as:
---- Notice ....
separated line ...
Last line -----

With the changed proposed, the blockquotes seems to be working fine with new lines inside.